bedrock/press/templates/press/speaker-request.html

```diff
@@ -1,274 +1,279 @@
+        {{ form.sr_event_audience_size }}
+      </li>
+
+      <li>
+        {{ form.sr_event_audience_demographics.errors }}
+        <label for="sr_event_audience_demographics">{{ _('Audience Demographics') }}</label>
+        {{ form.sr_event_audience_demographics }}
+      </li>
+
+      <li>
+        {{ form.sr_event_speakers_confirmed.errors }}
+        <label for="sr_event_speakers_confirmed">{{ _('Confirmed Speakers') }}</label>
+        {{ form.sr_event_speakers_confirmed }}
+      </li>
+
+      <li>
+        {{ form.sr_event_speakers_invited.errors }}
+        <label for="sr_event_speakers_invited">{{ _('Invited Speakers') }}</label>
+        {{ form.sr_event_speakers_invited }}
+      </li>
+
+      <li>
+        {{ form.sr_event_speakers_past.errors }}
+        <label for="sr_event_speakers_past">{{ _('Past Speakers') }}</label>
+        {{ form.sr_event_speakers_past }}
+      </li>
+
+      <li>
+        {{ form.sr_event_media_coverage.errors }}
+        <label for="sr_event_media_coverage">{{ _('Media Coverage') }}</label>
+        {{ form.sr_event_media_coverage }}
+      </li>
+
+      <li>
+        {{ form.sr_event_sponsors.errors }}
+        <label for="sr_event_sponsors">{{ _('Event Sponsors') }}</label>
+        {{ form.sr_event_sponsors }}
+      </li>
+
+      <li>
+        {{ form.sr_event_confirmation_deadline.errors }}
+        <label for="sr_event_confirmation_deadline">{{ _('Confirmation Deadline') }}</label>
+        {{ form.sr_event_confirmation_deadline }}
+        <div class="field-note date-note">
+          {{ _('2015-03-14 or March 14, 2015') }}
+        </div>
+      </li>
+    </ol>
+  </fieldset>
+
+  <fieldset>
+    <legend>{{ _('Presentation Details') }}</legend>
+
+    <ol>
+      <li>
+        <label>{{ _('Type of Presentation') }}</label>
+        {{ form.sr_presentation_type }}
+      </li>
+
+      <li>
+        {{ form.sr_presentation_topic.errors }}
+        <label for="sr_presentation_topic">{{ _('Topic of Presentation') }}</label>
+        {{ form.sr_presentation_topic }}
+      </li>
+
+      <li>
+        {{ form.sr_presentation_length.errors }}
+        <label for="sr_presentation_length">{{ _('Expected Length') }}</label>
+        {{ form.sr_presentation_length }}
+        <div class="field-note">
+          ({{ _('in hours, e.g. 2.5') }})
+        </div>
+      </li>
+    </ol>
+  </fieldset>
+
+  <fieldset>
+    <legend>{{ _('Additional Information') }}</legend>
+
+    <div id="sr_attachment">
+      {{ form.sr_attachment.errors }}
+      <label for="sr_attachment">{{ _('PDF Attachment') }}</label>
+      {{ form.sr_attachment }}
+      <div class="field-note">
+        {{ _('Agenda, media list, etc.') }}
+        {{ _('Maximum file size is 5 MB.') }}
+      </div>
+    </div>
+  </fieldset>
+  <button type="submit" name="submit_form" class="mzp-c-button mzp-t-primary">{{ _('Submit') }}</button>
+</form>
+
+{% else %}
+<aside class="mzp-c-notification-bar mzp-t-success">
```

Comment:
This success page is also a little short and could do with a min-height: 
![image](https://user-images.githubusercontent.com/400117/69134954-3d77ba80-0ab0-11ea-9896-11ba3d9996e5.png)